### DISCUSSION TITLE
Ellipsis in facets with Vufind 9

### DISCUSSION THREAD

**Discussion by damien-git** [vufind-org/vufind#2965] [General]
This looks like a bug introduced with Vufind 9: long facet used to be truncated with an ellipsis at the end in Vufind 8. They no longer are in Vufind 9. The related CSS was moved from js-tree.less to bootstrap-custom.css. But the selector, ".jstree-facet li span.main", no longer applies as the new tree does not use the "main" class (it could be "text" instead).
I am not sure what it was for in the first place. The CSS mentions "Fix Firefox cutting the checkboxes", but I don't see any issue in Firefox, it just displays the long name instead of truncating it.
Anyway, I don't intend to propose a PR, and I am not sure how to fix it, but I thought I should mention it.

**Comment by demiankatz** [comment]
Thanks, @damien-git. If the commit mentions "cutting the checkboxes" it's possible this relates to a problem that only occurs when "orFacets" is turned on (which changes facet logic from AND to OR and puts little checkboxes next to all of the values). I think @EreMaijala might be the best qualified person to comment on this, but he's unfortunately (for us) on holiday until August. Perhaps @crhallberg might have some ideas in the meantime.

If you want to try changing the selector from main to text to see what happens, that might be an interesting data point (and you could turn on orFacets for the facet you're testing to see if the checkboxes get messed up or not). If you don't feel comfortable starting a PR, it might at least be worth opening a JIRA ticket so we can pin this bug to a target fix version -- I'm concerned if we leave this in discussions it will be forgotten about.

**Comment by damien-git** [comment]
Sorry, I should have created a JIRA instead. Here it is:
https://openlibraryfoundation.atlassian.net/browse/VUFIND-1621

**Reply by demiankatz** [reply]
Thanks, @damien-git, I'll get this on the roadmap to be fixed by release 9.1 (if not sooner).
